python/delta_sharing/tests/test_delta_sharing.py

```diff
@@ -648,7 +648,7 @@ def test_load_as_spark(
         spark = SparkSession.builder \
             .appName("delta-sharing-test") \
             .master("local[*]") \
-            .config("spark.jars.packages", "io.delta:delta-sharing-spark_2.12:0.6.0-SNAPSHOT") \
+            .config("spark.jars.packages", "io.delta:delta-sharing-spark_2.12:1.0.0-SNAPSHOT") \
```

Comment:
is this intentional?

Reply:
Yes, this actually should match the version number in version.sbt, to run local tests. I didn't do it right previously.